drivers/xaudio2/SCsub

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/python
+
+Import('env')
+
+env.add_source_files(env.drivers_sources, "*.cpp")
+env.Append(CXXFLAGS=['-DXAUDIO2_ENABLED'])
+env.Append(LINKFLAGS=['xaudio2_8.lib'])
```

Comment:
In `platform/winrt/detect.py` it seems that `LIBS += ["xaudio2"]` was sufficient, does it need to be that explicit for the linkflags? It won't be practical if we need to change the file everytime the soname/libname changes.